app/lib/antispam.rb

```diff
@@ -33,9 +33,10 @@ def initialize(status)
   end
 
   def local_preflight_check!(status)
-    return unless spammy_texts.any? { |spammy_text| status.text.include?(spammy_text) }
+    all_time_ban = all_time_spammy_texts.any? { |spammy_text| status.text.include?(spammy_text) }
```

Comment:
I am having trouble understanding the logic here.

Could we have it a bit clearer? Maybe with a function that returns true if the message needs to be blocked, that could look like:

```ruby
return all_time_ban || (recent_ban && suspicious_reply_or_mention?(status) && status.account.created_at >= ACCOUNT_AGE_EXEMPTION.ago)
```